tools/accuracy_checker/accuracy_checker/adapters/action_recognition.py

```diff
@@ -60,30 +92,59 @@ def validate_config(self):
         super().validate_config(on_extra_argument=ConfigValidator.WARN_ON_EXTRA_ARGUMENT)
 
     def configure(self):
-        self.priorbox_out = self.get_value_from_config('priorbox_out')
+        self.new_net = self.get_value_from_config('new_net')
         self.loc_out = self.get_value_from_config('loc_out')
         self.main_conf_out = self.get_value_from_config('main_conf_out')
         self.num_action_classes = self.get_value_from_config('num_action_classes')
         self.detection_threshold = self.get_value_from_config('detection_threshold')
-        add_conf_out_count = self.get_value_from_config('add_conf_out_count')
-        add_conf_out_prefix = self.get_value_from_config('add_conf_out_prefix')
         self.action_threshold = self.get_value_from_config('action_confidence_threshold')
         self.action_scale = self.get_value_from_config('action_scale')
-        if add_conf_out_count is None:
-            self.add_conf_outs = [add_conf_out_prefix]
-        else:
+        add_conf_out_prefix = self.get_value_from_config('add_conf_out_prefix')
+
+        if self.new_net:
```

Comment:
If we have so many changes to adopt the current adapter for new topology, maybe it makes sense to have separated adapter for this case?

Reply:
I'm not sure that in this way the choice of right adapter for the model will be obvious. Moreover, as I can see it doesn't fit into current set of adapters: single adapter for the network type.

Reply:
ok, agree with you